js/constants/AppConstants.js

```diff
@@ -23,3 +23,6 @@ export const RECALL_STAGES = {
   FIRST: 'recallFirstText',
   NONE : 'recallNoneText'
 };
+
+export const COOKIE_LAST_STATE = 'last_state';
```

Comment:
I'm going to change the cookie name to `memorize_ephesians_last_state` unless anyone has a better suggestion. Better to be safe lest we clobber another cookie on the vintagenc.com domain.